src/tests/Microsoft.PowerFx.Interpreter.Tests/IRTests.cs

```diff
@@ -64,6 +64,20 @@ public void RecordToRecordAggregateCoercionTest(string expr)
             Assert.IsNotType<ErrorValue>(result);
         }
 
+        [Theory]
+        
+        [InlineData("[{a:0},{a:\"3\",b:2}]", "Table({a:0,b:If(false,0)},{a:3,b:2})")]
```

Comment:
>[{a:0},{a:\"3\",b:2}] [](http://example.com/codeflow?start=21&length=21)

Is [] special?
Could we add a test where Expr uses Table()? 
like: 
Table({a:0},{a:\"3\",b:2})

Reply:
[] is an special case. Why? I need to understand this better with the PMs.
`Table({a:0},{a:\"3\",b:2})` gives a compilation error due to the conflict of types.